examples/module1/lesson1/validate-it/index.js

```diff
@@ -32,3 +23,49 @@ function validator() {
 }
 
 validator();
+
```

Comment:
Podejście klasowe bardzo spoko, też miałem je w głowie, aczkolwiek uznałem je za zbyt duże jak na potrzeby tego zadania.

Jedną uwagę jaką bym miał to sprzężenie errorów stricte z validatorami. Zmiana języka/internalization/lub sama zmiana treści errora łamię zasadę open/closed principle i wymaga edycji w samych validatorach.

Poza tym bardzo spoko. Może metoda isRequired jest lekko myląca, na pierwszy rzut oka można pomyśleć że to nie jest validator a funkcja sprawdzająca czy pole jest wymagane. Może isFilled?